src/main/java/no/nav/veilarbveileder/config/ApplicationConfig.java

```diff
@@ -23,15 +40,28 @@
 @EnableConfigurationProperties({EnvironmentProperties.class})
 public class ApplicationConfig {
 
+    public final static String APPLICATION_NAME = "veilarbveileder";
+
+    @Bean
+    public Credentials credentials() {
+        return getServiceUserCredentials();
+    }
+
     @Bean
     public Norg2Client norg2Client(EnvironmentProperties properties) {
         return new CachedNorg2Client(new NorgHttp2Client(properties.getNorg2Url()));
     }
 
     @Bean
-    public Pep veilarbPep(EnvironmentProperties properties) {
-        Credentials serviceUserCredentials = getServiceUserCredentials();
-        return new VeilarbPep(
+    public AxsysClient axsysClient(){
+        String url = (isProduction().orElse(false)) ? "https://axsys.intern.nav.no" : "https://axsys.dev.intern.nav.no";
```

Comment:
for her vil det ikke være mulig å bruke en service URL?

Reply:
Det er litt mer guffent å bruke service URLer mot tjenester utenfor PTO, siden vi ikke har så mye kontroll hvis de ønsker å bytte namespace eller cluster. Men man kunne egentlig brukt service URL i dette tilfellet

Reply:
Riktig. Vel den er jo uansett chached så det har jo ikke så mye å si

Reply:
Vi har helsesjekk på det, og axsys har byttet til team namespace så regner ikke med at det blir så mye endringer. Bytter til service url